helm/nats-operator/config/client-auth.json

```diff
@@ -0,0 +1,25 @@
+{
+  "users": [
+    {{- if and (.Values.cluster.auth.username) (not .Values.cluster.auth.users) }}
```

Comment:
should there be helm templating here?

Reply:
these are direct imports from the nats-io/nats-operator repo with little-to-no alteration.

the helm/nats-operator/templates/secret.yaml file inlines this config file then materializes the embedded helm-templates via `tpl`, eg: `clients-auth.json: {{ (tpl (.Files.Get "config/client-auth.json") . ) | b64enc }}`